aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/SRPHelper.kt

```diff
@@ -35,38 +34,10 @@ import org.jetbrains.annotations.TestOnly
  * https://github.com/aws-amplify/aws-sdk-android/blob/main/aws-android-sdk-cognitoidentityprovider/src/main/java/com/amazonaws/mobileconnectors/cognitoidentityprovider/CognitoUser.java#L3587
  * SRP requires Kotlin version 1.5+, and minSDK version 24.
  */
-class SRPHelper(private val username: String, private val password: String) {
+class SRPHelper(private val password: String) : AuthHelper() {
```

Comment:
Any reason why SRPHelper should extend AuthHelper?

Its usage is a bit confusing. Not clear whether to use SRPHelper or AuthHelper for secret hash since function is static to both.

Reply:
I have removed the extension. There is no need.